Backend/Controllers/CouponsController.cs

```diff
@@ -26,7 +26,13 @@ public CouponsController(ILogger<CouponsController> logger, ICouponRepository co
             _mapper = mapper;
         }
 
-        // GET: api/v1.0/coupons
+        /// <summary>
+        /// Retrieves all existing coupons.
+        /// </summary>
+        /// <returns>A list of all existing coupons.</returns>
+        /// <response code="200">Returns a list of existing coupons.</response>
+        /// <response code="404">If there are no coupons stored in the database.</response>
```

Comment:
Skip the "if"

Reply:
Goes for all places.
The "if" makes it hard to read the documentation. So in this case would I prefer:
No coupons found in the database.

Reply:
I just followed Microsofts way of commenting in their tutorial of implementing Swagger (https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-5.0&tabs=visual-studio). I was a bit hesitant to comment in "my own" way instead of their way. But I agree with you, it makes the documentation more readable if I remove all the "if"s in the comments. Changing it now and will commit when I am finished.